tasks/main.yml

```diff
@@ -23,6 +23,7 @@
   get_url:
     url: "https://archive.org/download/zeromq_{{ zeromq_version }}/zeromq-{{ zeromq_version }}.tar.gz"
     dest: "/tmp/zeromq-{{ zeromq_version }}.tar.gz"
+  validate_certs: no
```

Comment:
`validate_certs` is an option of `get_url`. It should be indented 2 more spaces.